distributed/utils_test.py

```diff
@@ -1780,7 +1781,29 @@ def clean(threads=True, instances=True, processes=True):
 
 @pytest.fixture
 def cleanup():
```

Comment:
we're going to break everyone who uses this fixture anyway, so we can just use the fact that independent fixtures need importing even if they are added as simple dependencies to get the news out

```suggestion
@pytest.fixture
def cleanup(gonna_run_dask):
    # cleanup is deprecated in favor of gonna_run_dask see https://github.com/dask/distributed/pull/6854/
    return gonna_run_dask
```